### ISSUE TITLE
Humans with tails don't get tail organs

### ISSUE THREAD

**Issue by PersianXerxes** [TyrianTyrell/PampieStationRecoded#48]
## Round ID:
N/A

## Testmerges:

N/A

## Reproduction:

A human character with a tail selected doesn't spawn with a tail "organ"
This is a problem for future Persi to fix, somehow...

**Comment by WanderingFox95** [comment]
this is true for anything other than felines and lizard-people by the way.
Citadel's code makes tails a visual thing, not an organ thing.
This is why the cat surgeon only half-works on people sometimes.
It puts a cat tail in the organ slot but the tail the person picked isn't actually real and overlaps.
